charts/stable/paperless-ng/values.yaml

```diff
@@ -1,18 +1,51 @@
 image:
-  repository: tccr.io/truecharts/paperless-ng
+  repository: jonaswinkler/paperless-ng
   pullPolicy: IfNotPresent
-  tag: v1.5.0
+  tag: v1.5.0@sha256:e4102d1b850c9a54f7ac1f3a0be197f451ad36ae165b43aa9829714e1d582ca8
```

Comment:
you'll have to edit the container repo afterwards, replace this in the dockerfile and reset this to tccr.

Reply:
yep!